snap/snapcraft.yaml

```diff
@@ -208,6 +207,8 @@ parts:
       # Build the redis PHP module
       - source: https://github.com/phpredis/phpredis/archive/3.1.4.tar.gz
         source-checksum: sha256/656cab2eb93bd30f30701c1280707c60e5736c5420212d5d547ebe0d3f4baf71
+      # Build the mcrypt PHP module  
+      - source: https://github.com/php/pecl-encryption-mcrypt.git
```

Comment:
Can you please change this to a specific release, along with a checksum, please?

Reply:
I changed now the source to the latest commit and added checksum verification.